formidable/serializers/defaults.py

```diff
@@ -17,7 +18,16 @@ class Meta:
         fields = ('value',)
 
     def to_internal_value(self, data):
-        return {'value': data}
+        # NOTE: for some reason, data equals to the value.
+        # FIXME: investigate a bit further why.
```

Comment:
Do you want to do it in this PR or it's a reminder for later?

Reply:
I'm really not sure... it goes beyond the scope of this PR, and I wish I knew better about DRF to figure out why we have to convert to dict. As you can see, on master, it's already converted in the `to_internal_value` method.